lib/backup/syncer/rsync/base.rb

```diff
@@ -8,12 +8,14 @@ class Base < Syncer::Base
         ##
         # Additional String or Array of options for the rsync cli
         attr_accessor :additional_rsync_options
+        attr_accessor :disable_archive_option
 
         def initialize(syncer_id = nil, &block)
           super
           instance_eval(&block) if block_given?
 
           @path ||= '~/backups'
+          @disable_archive_option ||= false
```

Comment:
Can you rename this to `archive_option`? All the other options are also named in the "positive" way. Following the true path, so we don't have to deal with double negatives. Should make it easier for the end user as well.

Its default can then be true.